Packs/FeedTAXII/ReleaseNotes/1_1_32.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### TAXII 2 Feed
+
+- Updated the integration to fetch all available object types instead of a predefined list of object types in the case that *STIX Objects To Fetch* parameter is not set.
```

Comment:
```suggestion
- Updated the integration to fetch all available object types instead of a predefined list of object types when *STIX Objects To Fetch* parameter is not set.
```